### DISCUSSION TITLE
Centralized services

### DISCUSSION THREAD

**Discussion by ziiw** [bluesky-social/atproto#953] [Clarifications and "Why?"] [answered]
New here, discovering the protocol,

What are the parts of this protocol that are common for multiple clients?
I'm not sure to understand if there are centralized building blocks, eg: resolving user identities

If others client than Bluesky app are created on top of the ATProtocol, will they be able to interoperate without touching any of the Bluesky company own network / services?

Thanks !

**Comment by ianklatzco** [comment, marked as answer]
> What are the parts of this protocol that are common for multiple clients

atproto.com

> centralized building blocks, eg: resolving user identities

https://atproto.com/specs/did-plc

> will they be able to interoperate without touching any of the Bluesky company own network / services

Once federation is out, probably. It is currently WIP and the highest priority since blocking was finished.
![image](https://user-images.githubusercontent.com/1487413/235678451-53a99d48-74d0-422f-b4e0-67daf70020b7.png)

**Reply by dholms** [reply]
yup yup that covers it & just to clarify: we have plans to make did:plc actually decentralized as well & will be treating it as immutable public infrastructure until that happens
